runtime/gc_modron_startup/mminit.cpp

```diff
@@ -492,6 +492,12 @@ j9gc_initialize_heap(J9JavaVM *vm, IDATA *memoryParameterTable, UDATA heapBytesR
 	}
 #endif
 
+#if defined(J9VM_ARCH_X86)
+	if (J9_ARE_ANY_BITS_SET(vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_ENABLE_PORTABLE_SHARED_CACHE)) {
+		extensions->shouldForceLowMemoryHeapCeilingShiftIfPossible = true;
+	}
+#endif /* defined(J9VM_ARCH_X86) */
```

Comment:
it is much better but I still don't like `#if defined(J9VM_ARCH_X86)` here. This is not clear connection why shift should be forced for x86 only (something else hidden here happen somewhere, right?) I believe would be better to have global feature control boolean and use it here instead of `true`. Or you can just set bit for x86 only if it is fits better to your logic.

Reply:
the rational for this when you need to expand the feature to another platform would be no need to modify this code again... Ideally should be a single place single boolean to control enabled/disabled platforms

Reply:
Also is it correct place to set variable? Looks like it is too late... Should be set at the beginning of this function before line 321:
```
    extensions->heap = extensions->configuration->createHeap(&env, heapBytesRequested);
```

Reply:
Okay I agree, it makes more sense to set `J9_EXTENDED_RUNTIME2_ENABLE_PORTABLE_SHARED_CACHE` for x86 only.